src/sentry/options/defaults.py

```diff
@@ -411,6 +411,13 @@
     flags=FLAG_ALLOW_EMPTY | FLAG_PRIORITIZE_DISK | FLAG_AUTOMATOR_MODIFIABLE,
 )
 
+# Taskbroker
+register(
+    "taskbroker.enabled",
```

Comment:
This is a good solution for now. When we have taskbroker running in production, it would be good to tie into the `--workers` option for `sentry devserver` so that starting taskworkers is as simple as celery.

Reply:
I'm going to leave this for now, since using --workers requires creating a new `sentry run ...` command.